tests/test_sockets.py

```diff
@@ -1002,6 +1002,24 @@ async def test_cannot_connect(self, socket_path: Path) -> None:
         with pytest.raises(FileNotFoundError):
             await connect_unix(socket_path)
 
+    async def test_connecting_using_bytes(
+        self, server_sock: socket.socket, socket_path: Path
+    ) -> None:
+        async with await connect_unix(str(socket_path).encode()):
+            pass
+
+    @pytest.mark.skipif(
+        sys.platform == "darwin", reason="OSX requires valid UTF-8 paths"
```

Comment:
Other AnyIO skip markers use `platform.system()`; this should remain consistent.
```suggestion
        platform.system() == "Darwin", reason="macOS requires valid UTF-8 paths"
```

Reply:
For what it's worth, I copied it from the one immediately below which uses ``sys.platform`` :)

Reply:
Oh, that's on me then :)
All the newer skip marks use `platform.system()`.